bip-newproposal.mediawiki

```diff
@@ -0,0 +1,607 @@
+TBD | QES2 | A Hybrid Post-Quantum and Classical Digital Signature Scheme for Bitcoin | [Caleb Lee] director@j1729labs.online, [Justin Park] < justin7361@j1729labs.online>, [Eunice Lee] cuspro0103@j1729labs.online, [Sophia Shim] zypo1015@j1729labs.online | 2025-04-18 | Draft
+
+:BIP: Unassigned
+:Title: QES2 - A Hybrid Post-Quantum and Classical Digital Signature Scheme for Bitcoin
+:Author: [Caleb Lee] director@j1729labs.online, [Justin Park] justin7361@j1729labs.online, [Eunice Lee] cuspro0103@j1729labs.online, [Sophia Shim] zypo1015@j1729labs.online
+:Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+:Status: Draft
+:Type: Standards Track
+:Created: 2025-04-18
+:License: BSD-2-Clause
+:Requires: 340, 341, 342
```

Comment:
The document must start with the preamble adhering to the [required format](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#user-content-BIP_header_preamble). Beyond the formatting needing to be amended, the title is too long and the authors need to be on separate lines with the format `Name <addr@dom.ain>`.